.github/workflows/tests.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - run: npm ci
         # HACK: Force a TTY to enable browser-driver-manager to manipulate stdout.
       - shell: 'script -q -e -c "bash {0}"'
-        run: npx browser-driver-manager install chromedriver --verbose
+        run: npx browser-driver-manager install chromedriver=119 --verbose
```

Comment:
This is a temporary fix that will be fix in a separate pr to fix the cli tests with 120 https://github.com/dequelabs/axe-core-npm/issues/963.